resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                    <td>
+        <pre class="code--block">
+        {
+          "accountClosureReason": "All funds withdrawn",
+          "closureDate": "2017-05-20"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNTID_NOT_FOUND",
+          "message": "Enter a real accountId"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request with an invalid 'Accept' header</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br>accountId: 1234567890<br><br>Accept: application/vnd.hmrc.1.0</p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountClosureReason": "All funds withdrawn",
+          "closureDate": "2017-05-20"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "MATCHING_RESOURCE_NOT_FOUND",
+          "message": "A resource with the name in the request can not be found in the API"
+        }
+        </pre>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+      operationId: closeExistingLisaAccount
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/contentTypeHeader'
+      - $ref: '#/components/parameters/authorizationHeader'
+      - $ref: '#/components/parameters/lisaManagerReferenceNumber'
+      - $ref: '#/components/parameters/accountId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/lifetimeIsaManagerAccountsCloseAccountRequest'
+            examples:
+              example-1:
+                value:
+                  accountClosureReason: All funds withdrawn
+                  closureDate: 2017-05-20
+              example-2:
+                value:
+                  accountClosureReason: Cancellation
+                  closureDate: 2017-05-20
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/lifetimeIsaManagerAccountsCloseAccountResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                badRequest:
+                  $ref: '#/components/examples/invalidLmrn'
```

Comment:
This has been duplicated, please add the correct reference: `'#/components/examples/badRequest'`